app/src/main/kotlin/com/wire/android/core/di/Injector.kt

```diff
@@ -0,0 +1,35 @@
+package com.wire.android.core.di
+
+import android.content.Context
+import com.wire.android.feature.auth.registration.di.registrationModule
+import org.koin.android.ext.koin.androidContext
+import org.koin.core.context.startKoin
+import org.koin.core.module.Module
+
+object Injector {
+
+    private val coreModules: List<Module> = listOf()
+
+    /**
+     * Shared modules should contain dependencies that can
+     * build up multiple features
+     */
+    private val sharedModules: List<Module> = listOf()
+
+    /**
+     * Feature modules should contain dependencies that build up specific
+     * features and don't tend to live outside of that feature
+     */
+    private val featureModules: List<Module> = listOf(registrationModule)
+
+    fun start(context: Context) {
+        startKoin {
```

Comment:
As I pointed out in the other PR with the styles: Brackets {} are fine to keep them this way in order to favor the readability of the expression, but the arguments...mmm...I do not see it and I think we are not writing concise Kotlin code. This is how it would look for me:

```kotlin
        startKoin {
            androidContext(context)
            modules(listOf(coreModules, sharedModules, featureModules).flatten())
        }
 ```

I would also be of the idea of extracting the list of modules to a constant.

Reply:
combined them into a single line. I didn't feel the need to create a new variable for the module list. If we have one more in the future, I can add it.